login.css

```diff
@@ -22,7 +22,7 @@ body{
 
 nav{
 	width:100%;
-	background-color:#C8A2C8;
+	background-color:#0C59CF;
```

Comment:
06/08/2021: I change the color theme for the nav bar to navy blue. Because through out some testing I decided it would be a color that will have to give a great contrast and potentially make it easier for user to read and received information. Also I think the color combination two is match match will my music database.